src/psyclone/dynamo0p3.py

```diff
@@ -4449,10 +4472,10 @@ def gen_code(self, parent):
         depending on the loop type and then call the base class to
         generate the code.
 
-        :param parent: an f2pygen object that will be the parent of
+        :param parent: an f2pygen object that will be the parent of \
         f2pygen objects created in this method
         :type parent: :py:class:`psyclone.f2pygen.BaseGen`
-        :raises GenerationError: if a loop over colours is within an
+        :raises GenerationError: if a loop over colours is within an \
```

Comment:
OK, I have to ask. What's with the '\' - are new versions of pycodestyle or pylint complaining?

Reply:
I learnt this from Joerg - without it the generated documentation for the interface is not formatted correctly.